src/DependencyInjection/Configuration.php

```diff
@@ -67,13 +67,13 @@ public function getConfigTreeBuilder(): TreeBuilder
                             ->defaultValue(interface_exists(NormalizerInterface::class))
                         ->end()
                         ->scalarNode('default_region')
-                            ->defaultValue(PhoneNumberUtil::UNKNOWN_REGION)
+                            ->defaultValue(RegionCode::ZZ)
```

Comment:
The class RegionCode should not be used, it's internal and do not contains all region code actually supported.

Reply:
https://github.com/giggsey/libphonenumber-for-php/commit/b01b0b678dd90367599a55993eae950212f1be46 PhoneNumberUtil::UNKNOWN_REGION is now public again, so this change can be reverted (including all others below)